.github/workflows/build.yaml

```diff
@@ -27,9 +27,20 @@ jobs:
       - name: Update pip
         run: pip3 install pip --upgrade
 
-      - name: Install pip modules
+      - name: Install python3.8 modules
+        if: ${{ matrix.shortcut == 'cs8' }}
         run: >
-          pip3 install pycodestyle pylint==2.4.4 voluptuous yamllint "rstcheck<3.5.0" antsibull-changelog "rich<11.0.0" "ansible-lint<5.0.0" 'cryptography<37.0.0'
+          pip-3.8 install pycodestyle pylint==2.4.4 voluptuous yamllint "rstcheck<3.5.0" antsibull-changelog "rich<11.0.0" "ansible-lint<5.0.0" 'cryptography<37.0.0'
```

Comment:
Also once we moved to Python 3.8, do we really need those version limitation around cryptography, rstcheck, ...?

Reply:
good point, will remove it